doc/cygnus-ngsi/installation_and_administration_guide/running_as_process.md

```diff
@@ -21,4 +21,4 @@ The parameters used in these commands are:
 * `-p` (or `--mgmt-if-port`). Configures the listening port for the Management Interface. If not configured, the default value is used, `8081`.
 * `-g` (or `--web-app-port`). Configures the port where the web application for Cygnus runs. If not configured, the default value is used, `8082`.
 * `-t` (or `--polling-interval`). Configures the polling interval (seconds) when the configuration is periodically reloaded. If not configured, the default value is used, `30`.
-* `--no-yafs`. Disables Cygnus' YAFS (Yet Another Flume Supervisor), a supervisor in charge of starting an ordered stop of Cygnus when some thread is found dead.
+* `--no-yafs`. Disables Cygnus' [YAFS (Yet Another Flume Supervisor)](../../cygnus-common/installation_and_administration/yafs.md), a supervisor in charge of starting an ordered stop of Cygnus when some thread is found dead.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1439/commits/34f35f5fe3021ab8cfb43b7f0dc77f3f0009b882